confluent_kafka/avro/__init__.py

```diff
@@ -26,7 +26,10 @@ class AvroProducer(Producer):
     """
 
     def __init__(self, config, default_key_schema=None,
-                 default_value_schema=None, schema_registry=None):
+                 default_value_schema=None,
```

Comment:
may not be ideal, but it's better to not rearrange the order of existing arguments since users could already be calling this without using keyword arguments